src/bin/dolos/init/include/mod.rs

```diff
@@ -36,3 +37,31 @@ pub fn save_genesis_configs(root: &Path, network: &super::KnownNetwork) -> miett
 
     Ok(())
 }
+
+pub fn network_mutable_slots(network: &super::KnownNetwork) -> u64 {
+    let genesis = match network {
+        super::KnownNetwork::CardanoMainnet => Genesis {
+            alonzo: serde_json::from_slice(mainnet::ALONZO).unwrap(),
+            conway: serde_json::from_slice(mainnet::CONWAY).unwrap(),
+            byron: serde_json::from_slice(mainnet::BYRON).unwrap(),
+            shelley: serde_json::from_slice(mainnet::SHELLEY).unwrap(),
+            force_protocol: None,
+        },
+        super::KnownNetwork::CardanoPreProd => Genesis {
+            alonzo: serde_json::from_slice(preprod::ALONZO).unwrap(),
+            conway: serde_json::from_slice(preprod::CONWAY).unwrap(),
+            byron: serde_json::from_slice(preprod::BYRON).unwrap(),
+            shelley: serde_json::from_slice(preprod::SHELLEY).unwrap(),
+            force_protocol: None,
+        },
+        super::KnownNetwork::CardanoPreview => Genesis {
+            alonzo: serde_json::from_slice(preview::ALONZO).unwrap(),
+            conway: serde_json::from_slice(preview::CONWAY).unwrap(),
+            byron: serde_json::from_slice(preview::BYRON).unwrap(),
+            shelley: serde_json::from_slice(preview::SHELLEY).unwrap(),
+            force_protocol: Some(6),
+        },
+    };
+    ((3.0 * genesis.byron.protocol_consts.k as f32) / (genesis.shelley.active_slots_coeff.unwrap()))
+        as u64
```

Comment:
this is a ledger concern. We should have a function in ledger to compute the mutability window from a genesis object. Something [like this](https://github.com/txpipe/dolos/blob/ae1ade251a4ac446b81e8ad0d3074d343101696e/src/ledger/mod.rs#L251)